libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -425,6 +425,27 @@ export class BotFrameworkAdapter extends BotAdapter implements IUserTokenProvide
         return (await client.botSignIn.getSignInUrl(finalState, { channelId: context.activity.channelId }))._response.bodyAsText;
     }
 
+    /** 
+     * Retrieves the token status for each configured connection for the given user.
+     * @param context Context for the current turn of conversation with the user.
+     * @param userId The user Id for which token status is retrieved.
+     * @param includeFilter Optional comma seperated list of connection's to include. Blank will return token status for all configured connections.
+     * @returns Array of TokenStatus
+     * */ 
+    
+    public async getTokenStatus(context: TurnContext, userId?: string, includeFilter?: string ): Promise<TokenStatus[]>
```

Comment:
This is a [near perfect copy of the C# method](https://github.com/Microsoft/botbuilder-dotnet/blob/398ec3a235fd49a3295719b7d1ce07a6f5fe046d/libraries/Microsoft.Bot.Builder/BotFrameworkAdapter.cs#L665), so parity is already here (Good!):